contracts/core/extensions/CoreAdmin.sol

```diff
@@ -147,7 +147,7 @@ contract CoreInternal is
     {
         require(
             state.exchangeIds[_exchangeId] == address(0),
-            "Core: Invalid Id"
+            "Invalid Id"
```

Comment:
This feels hacky...I know we're trying to reduce bytecode size and this is a good way to do it but if that's the case we should change our error message return system wide. Cause some of these error messages are duplicates which almost defeats the purpose.

Reply:
+1. What's the thinking here? What is the difference in size from doing this? There are different ways to achieve this.

I will say that the current system doesn't provide all that much clarity either with duplicate `Core: Invalid Factory`. Some projects i've worked on have gone with a sequential bug identification i.e. `Core-1`, `Core-2`, etc. though with Core being so large, adding or removing a `require` could result in changes across multiple files. That's why we tried originally to do something like `Filename-MethodName-Error`